stories/core/Table.stories.tsx

```diff
@@ -861,13 +862,21 @@ export const LazyLoading: Story<Partial<TableProps>> = (args) => {
       emptyTableContent='No data.'
       onBottomReached={onBottomReached}
       isLoading={isLoading}
+      isSortable
       {...args}
       style={{ height: 440, maxHeight: '90vh' }}
       data={data}
+      // Prevents from resetting filters and sorting when more data is loaded
+      autoResetFilters={false}
```

Comment:
oh wow, interesting :O Wondering, if users were aware of these flags..

Reply:
We have a link to `react-table` documentation in Table description. But I doubt that a lot of users go there.

Reply:
Personally, I havent noticed these flags before.
Shouldnt be this a default value?

Reply:
Or maybe we could start adding these things to some wiki or doc.. 
Looks like a often used case and having these tips might be helpful :/

Reply:
I hope that our documentation site will have these examples.